templates/vue/src/store/getters.js

```diff
@@ -93,27 +93,30 @@ export function isVisible(state, { getNode, hasMultiContentAncestor }) {
   }
 }
 
-export function getActivities(state) {
-  return (options = {}) => {
-    const { exclude = [] } = options
-    return Object.values(state.nodes)
-      .filter(node => !exclude.includes(node.id) && Boolean(node.quiz))
-      .flatMap(node => node.quiz)
-  }
-}
-
 export function getQuestion(state) {
   return id => {
     const node = Object.values(state.nodes)
-      .filter(node => node.quiz)
-      .find(node => node.quiz.find(q => q.id == id))
+      .filter(node => node.typeData.activity)
+      .find(node => node.typeData.activity.questions.find(q => q.id == id))
     if (node) {
-      return node.quiz.find(q => q.id == id)
+      return node.typeData.activity.questions.find(q => q.id == id)
     }
     return null
   }
 }
 
+export function getAnswers(state) {
+  return (nodeId, questionId) => {
+    if (state?.userAnswers?.[nodeId]?.activity?.[questionId]) {
+      if (
+        typeof state.userAnswers[nodeId].activity[questionId].answers !== "string"
+      ) {
+        return state.userAnswers[nodeId].activity[questionId].answers
```

Comment:
Suggest using optional chaining syntax

Reply:
Hi @wynnset what would the optional chaining syntax be like here?

Reply:
`if (state?.userAnswers?.[nodeId]?.activity?.[questionId]?.answers !== "") {
      return state.userAnswers[nodeId].activity[questionId].answers
    } else {
      return {}
    }`